src/CidrValidator.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+namespace ipl\Validator;
+
+use ipl\I18n\Translation;
+use ipl\Stdlib\Str;
+
+/**
+ * Validate a classless inter-domain routing (CIDR)
+ */
+class CidrValidator extends BaseValidator
+{
+    use Translation;
+
+    public function isValid($value): bool
+    {
+        $this->clearMessages();
+
+        $pieces = Str::trimSplit($value, '/');
+        if (count($pieces) !== 2) {
+            $this->addMessage(sprintf($this->translate('CIDR "%s" is in the wrong format'), $value));
+
+            return false;
+        }
+
+        list($ip, $mask) = $pieces;
```

Comment:
`$address` and `$prefix` would be more precise. Messages should be adjusted accordingly.